Bottom: 06eec99a7c26e00fbcda748f75172c4e833faca1
Top:    1cc2646ded699ac4ef64b5a6de66269acd1f4acd
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-27 15:13:22 +0100

hid/gtk: Remove unnecessary prototype


---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index f5c836c..b1af557 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -32,8 +32,6 @@ RCSID ("$Id$");
 
 bool ghid_flip_x = false, ghid_flip_y = false;
 
-static void ghid_zoom_view_fit (void);
-
 static void
 pan_common (GHidPort *port)
 {
